final-project/manApiProxy.tex

```diff
@@ -4,23 +4,22 @@ \chapter{REST API Proxy \\
   \label{Chapter::RestApiProxy}}
 
 \section{Forwarding API Requests}
-When a REST API request is made to OpenTogetherTube, the request is first received by the load balancer. The 
-load balancer then selects one of the OTT monoliths based on the current load balancing algorithm, and forwards the 
-request to the selected monolith's nginx reverse proxy server. The nginx reverse proxy server receives the request and 
-forwards it to the corresponding monolith for processing. 
-The monolith processes the request and sends a response back to its nginx reverse proxy server. The reverse proxy server 
-then returns the response to the load balancer, which in turn sends the response back to the client that made the original
-request.
+When a REST API request is made to OpenTogetherTube, the request is first received by the load balancer
+ acting as the reverse proxy for API requests. The load balancer then selects one of the OTT monoliths based
```

Comment:
> The load balancer then selects one of the OTT monoliths based on the current load balancing algorithm and forwards the request to the selected monolith.

This is not true for all requests. For example, all endpoints that start with `/api/room/<roomname>` need to be routed to a specific monolith if the room is loaded.

I'm fine with merging this as is, as long as you open an issue saying this needs to be fixed.

Reply:
issued opened.